store.html

```diff
@@ -0,0 +1,144 @@
+                    <label class="label-padding" for="name">First name *</label>
+                    <input class="      " type="text" id="name" name="name">
+                </div>
+                <div class="display-flex-column">
+                    <label class="label-padding" for="last_name">Last name *</label>
+                    <input class="       " type="text" id="last_name" name="last_name">
+                </div>
+            </fieldset>
+            <fieldset class="display-flex-column">
+                <label class="label-padding" for="address">Address *</label>
+                <input type="text" id="address" name="address">
+            </fieldset>
+            <fieldset class="display-flex-column">
+                <label class="label-padding" for="address">Address 2 *</label>
+                <input type="text" id="address" name="address">
+            </fieldset>
+            <fieldset class="container-city-postcode">
+                <div class="city">
+                    <label class="label-padding" for="city">City *</label>
+                    <select class="select-city" name="city" id="city" required>
+                        <option value="London">Select your city...</option>
+                        <option value="London">London</option>
+                        <option value="Rome">Rome</option>
+                        <option value="Paris">Paris</option>
+                        <option value="Dublin">Dublin</option>
+                        <option value="Edinburgh">Edinburgh</option>
+                    </select>
+                </div>
+                <div class="postcode">
+                    <label class="label-padding" for="postcode">Postcode *</label>
+                    <input class="postcode-input" type="text" id="postcode" name="postcode">
+                </div>
+            </fieldset>
+            <fieldset>
+
+                <label class="label-padding" for="radio">Select a colour</label><br><br>
+                <input class="label-padding" type="radio" value="karmaorange" name="radio" id="karmaorange" required>
+                <label class="label-padding" for="karmaorange">Karma orange</label>
+                <input class="label-padding" type="radio" value="spacegrey" name="radio" id="spacegrey">
+                <label for="spacegrey">Space Grey</label><br><br><br><br>
+
+
+            </fieldset>
+            <fieldset>
+                <input id="checkbox" type="checkbox"/>
+                <label for="checkbox">By placing your order you agree to Karma's<span><a href="#" class="color-orange"> Terms and Conditions</a></span>.
+                    *</label>
+            </fieldset>
+            
+            <fieldset>
+                <button class="button-red-text-white">Place my order</button>
+            </fieldset>
+        </form>
+        <img class="image-girl-working" src="./additional/girl-working at desk.jpeg"> </img>
+
+
+    </main>
+    <footer>
+        <p>Join as on</p>
```

Comment:
A matter of style but I'd prefer an `h1` here as the main header for the footer. More semantically meaningful tag than generic `p`

Reply:
I've used already h1 in lane 51
Note: Only use one <h1> per page - this should represent the main heading/subject for the whole page. Also, do not skip heading levels - start with <h1>, then use <h2>, and so on.
 @kolomiets